Source/WebCore/platform/graphics/skia/PathSkia.cpp

```diff
@@ -164,9 +150,20 @@ void PathSkia::add(PathClosedArc closedArc)
     add(PathCloseSubpath());
 }
 
-void PathSkia::add(PathEllipse)
+void PathSkia::add(PathEllipse ellipse)
 {
-    notImplemented();
+    if (ellipse.rotation) {
+        AffineTransform transform;
+        transform.translate(ellipse.center.x(), ellipse.center.y()).rotateRadians(ellipse.rotation);
+        AffineTransform inverseTransform = transform.inverse().value();
```

Comment:
s/AffineTransform/auto/

Reply:
value_or(AffineTransform::identity) -- there are degenerate cases. maybe you even need to bail out for non-invertible case? value() is not safe here.

Reply:
I can add an assert, but the transform is always invertible, it's checked by the caller in`CanvasPath::ellipse()`